Dockerfile.windows

```diff
@@ -0,0 +1,20 @@
+FROM microsoft/dotnet:1.0.0-preview2-windowsservercore-sdk
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';"]
+
+# REMARK: Temporary workaround for Windows DNS client weirdness
+RUN set-itemproperty -path 'HKLM:\SYSTEM\CurrentControlSet\Services\Dnscache\Parameters' -Name ServerPriorityTimeLimit -Value 0 -Type DWord
+
+RUN New-Item -Path \MusicStore\samples\MusicStore.Standalone -Type Directory
+WORKDIR MusicStore
+
+ADD samples/MusicStore.Standalone/project.json samples/MusicStore.Standalone/project.json
+ADD NuGet.config .
+RUN dotnet restore --no-cache .\samples\MusicStore.Standalone
+
+ADD samples samples
+RUN dotnet build .\samples\MusicStore.Standalone
+
+EXPOSE 5000
+ENV ASPNETCORE_URLS http://0.0.0.0:5000
+CMD dotnet run -p .\samples\MusicStore.Standalone
```

Comment:
just tried to build with docker build (instead of compose), did I miss sth.?:   
(edit due to incorrect paste)
`Step 8/13 : RUN dotnet restore .\samples\MusicStore.Standalone
 ---> Running in 90630e2fe7d6
log  : Restoring packages for C:\MusicStore\samples\MusicStore.Standalone\project.json...
error: Unable to load the service index for source https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json.
error:   An error occurred while sending the request.
error:   The server name or address could not be resolved.`